src/tardev-snapshotter/src/snapshotter.rs

```diff
@@ -1012,16 +1012,79 @@ impl TarDevSnapshotter {
                         return Err(anyhow!(copy_error));
                     }
                 }
+                file.flush()?;
 
-                trace!("Appending index to {:?}", &base_name);
-                file.rewind().context("failed to rewind the file handle")?;
-                tarindex::append_index(&mut file).context("failed to append tar index")?;
+                // Create an erofs metadata using mkfs.erofs
+                let layer_path = base_name.with_extension("erofs");
+                debug!(
+                    "Creating erofs meta image {:?} from {:?}",
+                    &layer_path, &base_name
+                );
+                let status = Command::new("mkfs.erofs")
+                    .args([
+                        "--tar=i",
+                        "-T", "0", // zero out unix time
```

Comment:
aside from `--mkfs-time` option, `--aufs` option is also needed to convert OCI whiteouts/opaque to overlayfs metadata...

Reply:
Added --mkfs-time and --aufs